packages/libs/src/sdk/api/dashboard-wallet-users/types.ts

```diff
@@ -3,17 +3,31 @@ import { z } from 'zod'
 import { HashId } from '../../types/HashId'
 import { isEthAddressValid } from '../../utils/ethAddress'
 
-export const CreateDashboardWalletUser = z.object({
-  wallet: z.custom<string>((data: unknown) => {
-    return isEthAddressValid(data as string)
-  }),
-  userId: HashId,
-  walletSignature: z.object({
-    /** Message should be of the form: "Connecting Audius user id a93jl at 39823489" */
-    message: z.string(),
-    signature: z.string()
+export const CreateDashboardWalletUser = z
+  .object({
+    wallet: z.custom<string>((data: unknown) => {
+      return isEthAddressValid(data as string)
+    }),
+    userId: HashId,
+    walletSignature: z
+      .object({
+        /** Message should be of the form: "Connecting Audius user id a93jl at 39823489" */
+        message: z.string(),
+        signature: z.string()
+      })
+      .optional(),
+    userSignature: z
+      .object({
+        /** Message should be of the form: "Connecting Audius protocol dashboard wallet 0x6c9CA7D9580d4e8286B0628c0300A2A1235a8e2E at 39823489" */
```

Comment:
i think there is a way to z.string with more rules like starting with a string, worth adding to validation?